From 6b3b0adf22734c36f2e6d7fc98e7b78047aa8f2d Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Sun, 20 May 2018 14:38:31 -0700
Subject: [PATCH 8/8] build: Clean up makefile hook inclusions

* Include MTK/QCOM rules directly from
  vendor/lineage/build/core/config.mk hook, since that is what it is
  there for.
* Remove apicheck.mk hook, this is unused, and if it turns out to be
  needed again, it can be included by the config.mk hook mentioned above.
* Place main.mk hook into vendor/lineage/build/core/main.mk with the
  rest of the make build system hooks.

Change-Id: I51bda9f5d35a63113d6eab98e06da8919ba46480
---
 core/config.mk | 8 --------
 core/main.mk   | 2 +-
 2 files changed, 1 insertion(+), 9 deletions(-)

diff --git a/core/config.mk b/core/config.mk
index 00f0dde11..ea04c2e9d 100644
--- a/core/config.mk
+++ b/core/config.mk
@@ -989,14 +989,6 @@ $(eval include device/lineage/sepolicy/common/sepolicy.mk)
 # Include any vendor specific config.mk file
 -include $(TOPDIR)vendor/*/build/core/config.mk
 
-# Include any vendor specific apicheck.mk file
--include $(TOPDIR)vendor/*/build/core/apicheck.mk
-
-# Rules for QCOM targets
--include $(TOPDIR)vendor/lineage/build/core/qcom_target.mk
-
-# Rules for MTK targets
--include $(TOPDIR)vendor/lineage/build/core/mtk_target.mk
 endif
 
 include $(BUILD_SYSTEM)/dumpvar.mk
diff --git a/core/main.mk b/core/main.mk
index 27dbf5b3a..8f5e97ee5 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -327,7 +327,7 @@ ADDITIONAL_BUILD_PROPERTIES += dalvik.vm.stack-trace-dir=/data/anr
 
 # ------------------------------------------------------------
 # Include vendor specific additions to build properties
--include vendor/lineage/config/main.mk
+-include vendor/lineage/build/core/main.mk
 
 # ------------------------------------------------------------
 # Define a function that, given a list of module tags, returns
-- 
2.17.0

